src/QMCDrivers/DMC/DMCBatched.cpp

```diff
@@ -471,11 +511,13 @@ void DMCBatched::setMultiplicities(const DMCDriverInput& dmcdriver_input,
 void DMCBatched::runDMCStep(int crowd_id,
                             const StateForThread& sft,
                             DriverTimers& timers,
-                            //                            DMCTimers& dmc_timers,
+                            DMCTimers& dmc_timers,
                             UPtrVector<ContextForSteps>& context_for_steps,
                             UPtrVector<Crowd>& crowds)
 {
   Crowd& crowd = *(crowds[crowd_id]);
+  if (crowd.size() == 0)
```

Comment:
I'm cautious on such early return. Is it pure for performance or there is some bad behaviours when the size is zero?